policyengine_canada/parameters/gov/cra/tax/income/credits/gst_credit/singles_boost/singles_boost_multiplier.yaml

```diff
@@ -0,0 +1,9 @@
+description: GST credit singles boost multiplier
```

Comment:
Replace with a scale parameter called `childless_phase_in.yaml` combining the 2% with the $9,919 threshold